py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -351,6 +351,34 @@ def aten_ops_softmax(
     )
 
 
+@dynamo_tensorrt_converter(
+    torch.ops.aten.split.Tensor, capability_validator=dynamic_unsupported_with_args([1])
```

Comment:
The `dynamic_unsupported_with_args` import may also have been removed in the rebase